app/model/Export/templates/participantEducation.latte

```diff
@@ -32,19 +32,35 @@ td {
 
 <table class="all" border="1">
     <tr>
-        <th style="width: 5%;">P.č.</th>
```

Comment:
ty procenta zajistovala aby se to nejak rozumne vykreslilo a zalomilo

Reply:
No já jsem měl spíš pocit, že se pak zbytečně zalomovaly řádky když se nějaký sloupec nevešel, i když v ostatních bylo místo... Tak mi přišlo lepší to nechat dynamický

Reply:
Ok klidně to zde zkus